README.md

```diff
@@ -234,8 +242,21 @@ Please check out common DSC Resources [contributing guidelines](https://github.c
 ### Unreleased
 
 * xSQLServerSetup:
-	- Corrected bug in GetFirstItemPropertyValue to correctly handle registry keys with only one value.
->>>>>>> cd039bfd72f5a433a527cef05634382050a4e356
+  - Corrected bug in GetFirstItemPropertyValue to correctly handle registry keys with only one value.
+  - Added support for SQL Server 2008 R2 installation
+  - Removed default values for parameters, to avoid compatibility issues and setup errors
+  - Added Replication sub feature detection
+  - Added setup parameter BrowserSvcStartupType
+* xSQLServerFirewall
+  - Removed default values for parameters, to avoid compatibility issues
+  - Updated firewall rule name to not use 2012 version, since package supports 2008, 2012 and 2014 versions
+* xSQLServerNetwork

```

Comment:
Can you add test coverage or some E2E example for the new xSQLServerNetwork resource?